lib/scope.js

```diff
@@ -991,13 +994,78 @@ module.exports = {
 
         await scope.afterStep(scope, { waitForShowIf: true });
         
+      } else if ( type === `file` ) {
+        await scope.uploadFiles( scope, { handle, set_to });
+        await scope.afterStep(scope, { waitForShowIf: true });
+
       } else {
         await scope.setText( scope, { handle, set_to });
-        await scope.afterStep(scope );  // No showifs for this one?
+        await scope.afterStep(scope, { waitForShowIf: true });
       }
     },  // Ends scope.setField.input()
   },
 
+  uploadFiles: async function ( scope, { handle, set_to }) {
+    /* Try to upload one or more files to a file-type input field. */
+
+    // Allow a typo where someone forgot to include a space after a comma?
+    let file_names = set_to.split(`,`);
```

Comment:
In my head, this line belongs closer to the for loop below where we use the `file_names` variable for the first time.